packages/shared/utils/svgTransform.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     const code = babel.transform(
       `
         import React from 'react';
-        export default () => (<svg data-filename="${path.relative(process.cwd(), filename)}" />);
+        export default () => (<svg data-filename="${path.relative(process.cwd(), filename).replace(/\\/g, '/')}" />);
```

Comment:
This was another source of failure for tests, since windows uses `\\` file path delimiters. Though, this didn't seem to make a difference. I thought based on the jest config this was where `svg`s were being transformed for tests, is there somewhere else to look at?

Reply:
Did the snapshot tests fail due to a missmatch in the SVG delimiter in windows?

Reply:
yup